frontend/src/pages/ProfileNavigation.js

```diff
@@ -0,0 +1,553 @@
+      <DisplayAlbums/>
+
+      <AddAlbumModal
+        show={modalShow}
+        onHide={() => setModalShow(false)}
+      />
+    </Col>
+  )
+}
+
+/** Create new album modal
+ *   props: show, onHide
+ */
+function AddAlbumModal(props) {
+  return (
+    <Modal
+      {...props}
+      size="lg"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered
+    >
+      <Modal.Header closeButton>
+        <Modal.Title id="contained-modal-title-vcenter">
+          Create Album
+        </Modal.Title>
+      </Modal.Header>
+      <Modal.Body>
+        <Container>
+          <Form onSubmit={async (e) => {
+            e.preventDefault();
+            // following this: https://stackoverflow.com/questions/63182107/react-bootstrap-get-value-from-form-on-submit
+            const formData = new FormData(e.target),
+              formDataObj = Object.fromEntries(formData.entries())
+            await fetch('/api', {
+              method: 'POST',
+              headers: {
+                'Content-Type': 'application/json',
+              },
+              body: JSON.stringify({ action: 'createAlbum', ...formDataObj })
+            })
+              .then(res => res.json())
+              .then(data => window.location.reload())
+              .catch(console.log)
+          }} >
+            {/* Album name */}
+            <Form.Group controlId="name" >
+              <Form.Label>Album name</Form.Label>
+              <Form.Control 
+              as="textarea" 
+              rows="1" 
+              placeholder="Enter album name" 
+              name='name' />
+            </Form.Group>
+
+            {/* Album description */}
+            <Form.Group controlId="description">
+              <Form.Label>Description</Form.Label>
+              <Form.Control 
+              as="textarea" 
+              rows="3" 
+              placeholder="Enter album description" 
+              name='caption' />
+            </Form.Group>
+
+            {/* Album privacy */}
+            <Form.Group controlId="privacy">
+              <Form.Label>Privacy</Form.Label>
+              <PrivacySettingToggle />
+            </Form.Group>
+
+            {/* Buttons */}
+            <Row style={{ justifyContent: "flex-end" }}>
+              <Button
+                onClick={props.onHide}
+                variant="light"
+                style={{ marginRight: "1em", color: "grey" }}>
+                Cancel
+                </Button>
+              <Button
+                type='submit'
+                variant="light"
+                style={{ marginRight: "1em", borderColor: "grey", color: "grey" }}>
+                Submit
+                </Button>
+            </Row>
+          </Form>
+        </Container>
+      </Modal.Body>
+    </Modal>
+  );
+}
+
+/* Called by AddAlbumModal
+ * returns: a toggle button between private and public icons */
+function PrivacySettingToggle() {
+  const [checked, setChecked] = useState(false);
```

Comment:
This value does not seem to be in use. Are we leaving this here for future use or some other reason? There are also a few other values not used in this file that was outputted as a warning when compiling.

Reply:
I believe we are leaving this for the future.